src/Features/VisualBasic/Completion/KeywordRecommenders/Statements/ElseKeywordRecommender.vb

```diff
@@ -45,5 +45,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.KeywordRecommenders.Stat
 
             Return SpecializedCollections.EmptyEnumerable(Of RecommendedKeyword)()
         End Function
+
+        Private Function IsNotDirectlyInElseBlock(context As VisualBasicSyntaxContext) As Boolean
```

Comment:
Based on the current implementation, this method name seems wrong ("directness" isn't required)